include/deal.II/distributed/tria_base.h

```diff
@@ -40,8 +40,37 @@ namespace parallel
 {
   /**
    * This class describes the interface for all triangulation classes that
-   * work in parallel, namely parallel::distributed::Triangulation and
+   * work in parallel, namely parallel::distributed::Triangulation,
+   * parallel::fullydistributed::Triangulation, and
    * parallel::shared::Triangulation.
+   *
+   * It is, consequently, a class that can be used to test whether a
+   * pointer of reference to a triangulation object refers to a
+   * sequential triangulation, or whether the triangulation is in fact
+   * parallel. In other words, one could write a function like this:
+   * @code
+   *   template <int dim, int spacedim>
+   *   bool is_parallel (const dealii::Triangulation<dim,spacedim> &tria)
+   *   {
+   *     if (dynamic_cast<const
+   * parallel::TriangulationBase<dim,spacedim>*>(&tria)
```

Comment:
Would you mind adding some indentation in the comment here? (I believe the indentation script inserted the break here.)

Reply:
I'm never going to learn to love this tool... :-(